### PR TITLE
Add test for data persistency

### PR DESCRIPTION
**Pull request by arogachev**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

File: tests/Common/ManagerLogicTestTrait.php [hunk 1]
```diff
@@ -854,4 +859,39 @@ public function testRevokeAll(): void
...
+                'role2' => (new Role('role2'))->withCreatedAt(1_694_502_976)->withUpdatedAt(1_694_502_976),
+            ],
+            $itemsStorage->getAll(),
+        );
+        $this->assertEquals(
+            [
+                'role1' => (new Role('role1'))->withCreatedAt(1_694_502_936)->withUpdatedAt(1_694_502_936),
+                'role2' => (new Role('role2'))->withCreatedAt(1_694_502_976)->withUpdatedAt(1_694_502_976),
+            ],
+            $manager->getChildRoles('role1'),
```
Comment (vjik) [inline tests/Common/ManagerLogicTestTrait.php:887]: Child roles of `role1` must be `role2` only, right?

Comment (arogachev) [reply tests/Common/ManagerLogicTestTrait.php:887]: That's how it initially was. It needs to be fixed. I created issue earlier - #178.

Comment (vjik) [reply tests/Common/ManagerLogicTestTrait.php:887]: May be fix #178 before merge this PR. This suggestion for prevent create incorrect tests.

Comment (arogachev) [reply tests/Common/ManagerLogicTestTrait.php:887]: #191

**Review by vjik — APPROVED** [review]

**Pull request merged**